### PR TITLE
nrzled.NewSPI: also add padding in the front

### PR DESCRIPTION
**Pull request by flokli**
Some SPI devices take a while to set up for transfer, and while doing
so, show the first bit they're about to send.

This causes the first LED to always interpret the first symbol as a
logical 1. Zapping out the whole strip will leave the first LED green,
and the first pixel will always be slightly green-ish:

image

Prevent this, by applying the same 3 bit padding we have at the end to
the front:

image

### CODE REVIEW

**Comment by flokli** [comment]
Test failures seem unrelated:

> Run gosec -include=G104 -fmt=golint -quiet ./...
2022/05/14 12:15:59 internal error: package "errors" without types was imported from "command-line-arguments"
Error: Process completed with exit code 1.

**Comment by maruel** [comment]
Thanks! Will take a look in ~half a day. CI failures look unrelated, will take a look separately.

File: nrzled/nrzled_spi_test.go [hunk 1]
```diff
@@ -302,14 +311,17 @@ func TestSPI_Long(t *testing.T) {
 	if n, err := d.Write(toRGB(colors)); n != len(colors)*3 || err != nil {
 		t.Fatalf("%d %v", n, err)
 	}
-	expected := make([]byte, 12*o.NumPixels+3)
-	for i := 0; i < 12*o.NumPixels; i += 12 {
+	expected := make([]byte, 3+12*o.NumPixels+3)
+	expected[0] = 0x00
```
Comment (maruel) [inline nrzled/nrzled_spi_test.go:315]: Remove these, they are guaranteed to be zero.

Comment (flokli) [reply nrzled/nrzled_spi_test.go]: done. I made a comment, to explain why the loop starts with `i = 3`

**Comment by maruel** [comment]
I fixed CI. I'll still squash since I can't run the checks on tip of tree.

**Comment by maruel** [comment]
https://github.com/periph/devices/actions/runs/2340193976 worked!

**Comment by flokli** [comment]
🎉

**Comment by maruel** [comment]
Released v3.6.14: https://github.com/periph/devices/releases/tag/v3.6.14

I was long overdue.

**Comment by maruel** [comment]
ugh, forgot to git fetch before tagging, let me create v3.6.15 :(

**Pull request merged**